pages/test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Passports from '@/components/Passports'
```

Comment:
when I go to `localhost:3000/test`, I see 
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/62922309/228150462-81083e92-ff73-4082-aaf5-973f6dbd3734.png">

it doesn't seem like the pfps flip like the ones you showed me before